### PR TITLE
Fix code style

### PR DESCRIPTION
**Pull request by sektor-sumy**
Hey!

Type: code quality

Link to issue:

**In raising this pull request, I confirm the following (please check boxes):**

- I have read and understood the [Contributing Guidelines](https://github.com/SocialConnect/auth/blob/master/.github/CONTRIBUTING.md).
- I have checked that another pull request for this purpose does not exist.

Small description of change:
Hello I was sad to look at the code and I decided to put it in order a bit
Thanks :smiley_cat:

### CODE REVIEW

**Comment by heiglandreas** [comment]
Thanj you! It looks like your change broke CI. And especially the CodeSniffer that ensures the Coding Style adheres to PSR-2.

**Comment by sektor-sumy** [comment]
sorry, fixed)

**Comment by sektor-sumy** [comment]
> I personally find the moving of method locations unnecessary and noisy.

I think everything should be in place
it is very strange to see public methods then protected and then again public

File: src/OAuth1/AbstractProvider.php [hunk 1]
```diff
@@ -18,6 +18,9 @@
 use SocialConnect\OAuth1\Signature\MethodHMACSHA1;
 use SocialConnect\Provider\Session\SessionInterface;
 
+/**
+ * Class AbstractProvider
```
Comment (ovr) [inline src/OAuth1/AbstractProvider.php]: Please remove this comments (for classes), it's not needed due useless

File: src/OAuth1/AccessToken.php [hunk 2]
```diff
@@ -8,10 +8,13 @@
 
 use SocialConnect\Provider\AccessTokenInterface;
 
+/**
+ * Class AccessToken
```
Comment (ovr) [inline src/OAuth1/AccessToken.php]: Please remove this comments (for classes), it's not needed due useless

File: src/OAuth1/Exception/UnknownAuthorization.php [hunk 3]
```diff
@@ -6,8 +6,15 @@
 
 namespace SocialConnect\OAuth1\Exception;
 
+/**
+ * Class UnknownAuthorization
```
Comment (ovr) [inline src/OAuth1/Exception/UnknownAuthorization.php]: Please remove this comments (for classes), it's not needed due useless

**Comment by ovr** [comment]
Hey!

LGTM, but please remove all added comments to classes, because it's not needed

Thanks

**Pull request closed without merging**